### DISCUSSION TITLE
[General] <Questions about zksync's connectivity>

### DISCUSSION THREAD

**Discussion by kientt265** [zkSync-Community-Hub/zksync-developers#715] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hello everyone, I'm a web3 developer who just joined the community. I have a question: can zkSync connect to my local blockchain system? I'm wondering if I need to connect my system to ethereum because some of the resources I found were uncertain about this.

**Comment by alhenaai** [comment]
Hello! Welcome to the ZKsync community. Yes, ZKsync can connect to your local blockchain system. You can set up a local ZK chain and use ZKsync CLI to interact with it, specifying RPC URLs for both L1 (your local blockchain) and L2 (ZKsync). This setup allows you to develop and test applications in a controlled environment without needing to connect to the Ethereum mainnet.

For more detailed guidance on setting up and using a local ZK chain, you can refer to the [Using a Local ZK Chain documentation](https://docs.zksync.io/zk-stack/running-a-zk-chain/using-a-local-zk-chain). This resource provides comprehensive instructions on configuring and interacting with your local setup.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by iscxyz** [comment, marked as answer]
Hello, thank you for your question! For general information and recommendations, please refer to the [zkSync documentation](https://docs.zksync.io/). If you come across a specific issue, don’t hesitate to open a new ticket—we’re here to help.

I’ll go ahead and close this ticket for now.
